src/wpsmon.c

```diff
@@ -57,6 +57,9 @@ static struct mac {
 enum seen_flags {
 	SEEN_FLAG_PRINTED = 1,
 	SEEN_FLAG_COMPLETE = 2,
+	SEEN_FLAG_PBC = 4,
+	SEEN_FLAG_LOCKED = 8,
+	SEEN_FLAG_WPS_ACTIVED = 16,
```

Comment:
"actived" isn't english. do you mean active, or activated ?

Reply:
activated :sweat_smile: